cmd/minikube/cmd/start.go

```diff
@@ -87,8 +86,9 @@ var startCmd = &cobra.Command{
 
 // Set default value for host data and config dir
 var (
-	hostConfigDirectory = "/var/lib/minishift/openshift.local.config"
-	hostDataDirectory   = "/var/lib/minishift/hostdata"
+	hostConfigDirectory  = "/var/lib/minishift/openshift.local.config"
+	hostDataDirectory    = "/var/lib/minishift/hostdata"
+	hostVolumesDirectory = "/var/lib/minishift/openshift.local.volumes"
```

Comment:
@hferentschik I will #409 later as this PR is critical for next release and I do not want to delay it.